File: Bars.GkhDI/Entities/Service/CostItem.cs
namespace Bars.GkhDi.Entities
{
    using Gkh.Entities;

    /// <summary>
    /// Статьи затрат
    /// </summary>
    public class CostItem : BaseGkhEntity
    {
        /// <summary>
        /// Базовая услуга
        /// </summary>
        public virtual BaseService BaseService { get; set; }

        /// <summary>
        /// Наименование статьи
        /// </summary>
        public virtual string Name { get; set; }

        /// <summary>
        /// Количество
        /// </summary>
        public virtual decimal? Count { get; set; }

        /// <summary>
        /// Сумма
        /// </summary>
        public virtual decimal? Sum { get; set; }

        /// <summary>
        /// Цена за еденицу
        /// </summary>
        public virtual decimal? Cost { get; set; }
    }
}

File: Bars.GkhDI/libs/B4/model/Service/CostItem.js
Ext.define('B4.model.service.CostItem', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'CostItem'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'BaseService', defaultValue: null },
        { name: 'Name' },
        { name: 'Count', defaultValue: null },
        { name: 'Sum', defaultValue: null },
        { name: 'Cost' }
    ]
});